### PR TITLE
Package server

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: server/geode_server_protocols.py [hunk 1]
```diff
@@ -31,132 +43,9 @@ def initializeProtocols(self):
...
-        print(OpenGeodeServerProtocol.modules)
-        for module_name in OpenGeodeServerProtocol.modules:
-            module = import_module(module_name)
+        print(GeodeServerProtocol.modules)
+        for module_name in GeodeServerProtocol.modules:
+            module = import_module(module_name.strip())
             for module_protocol in module.protocols:
                 self.registerVtkWebProtocol(module_protocol())
 
-    def north(self):
```
Comment (panquez) [inline server/geode_server_protocols.py:40]: Has this code been removed or moved?

Comment (BotellaA) [reply server/geode_server_protocols.py:40]: Removed, the marker is now send from the front end.

Comment (panquez) [reply server/geode_server_protocols.py:40]: Ok

**Comment by BotellaA** [comment]
> LGTM after some cleaning

Cleaning will be done in a later PR. I need to test CD first but I need to me on the master...

**Comment by panquez** [comment]
> > LGTM after some cleaning
> 
> Cleaning will be done in a later PR. I need to test CD first but I need to me on the master...

Yes I agree. Ok so let's go !

**Pull request merged**